terraform/test/test.tfvars

```diff
@@ -0,0 +1,2 @@
+model   = "terraform"
+channel = "1.23/edge" # TODO: remove variable once 1.23/stable gets published
```

Comment:
I'd probably won't remove the variable. The default channel in Charmhub will change in the future, so: 1.23 -> 1.25 -> 1.2x. We can ensure that in this branch (track/1.23) we always test with the version that the branch points at.